docs/props/plugins.md

```diff
@@ -0,0 +1,59 @@
+# `plugins` prop
+
+## Overview
+
+The `plugins` property is optional and contains plugins for component.
+
+Plugin is a pure javascript function. At the moment available are two types of plugins: `mutation` and `extractor`. Their characteristic is described below.
+
+## Mutation plugin
+
+Task of mutation plugin is mutate content of specific type of document. 
+
+## Extractor plugin
+
+Task of extractor plugin is extract some additional data from content, like a metadata, headers (for `.md` files), etc. to `data` field of `Source` type.
+
+## Types
+
+For types of plugins, see [file](https://github.com/kyma-incubator/documentation-component/blob/master/packages/documentation-component/src/interfaces/Plugin.ts).
+
+## Passing globally options
+
+`plugins` property allows to define options for single plugin. Instead of passing a single one, pass object with `plugin` field (with appropriate plugin as value) and second `options` field. See example.
```

Comment:
```md
The `plugins` property allows also you to define options for a single plugin. Instead of passing a single one, pass an object with the **plugin** field with the appropriate plugin as value, and add the **options** field. 
```